### DISCUSSION TITLE
minimal and multi-page starter kit

### DISCUSSION THREAD

**Discussion by FredericHeem** [vanjs-org/van#42] [Show and tell]
I would like to present 2 different starter kits built on van.js:
* A minimal boiler plate built with vite, useful to play around, report minimal bug report, see https://github.com/FredericHeem/van-playground
* A complete starter kit supporting multi page app, nested routing with dynamic loading, theming, and css with goober, a 1Kb css in js solution, all under 5KB, see https://github.com/FredericHeem/van-kit

**Comment by Tao-VanJS** [comment]
🙏 Thank you so much! If you don't mind, I will consider linking your template somewhere from vanjs.org website and/or the README.md file. I believe the VanJS community will benefit greatly from your contribution!

**Reply by FredericHeem** [reply]
Thanks a lot for including the links.

**Reply by artydev** [reply]
Great thank you :-)
